### PR TITLE
created github workflow for mermaid diagrams

### PR DESCRIPTION
**Pull request by elf-pavlik**
Mermaid diagrams will be generated for all `*.mmd` files and included in `gh-pages` deployment.

* [x] github workflow
* [x] section in README
* [x] .gitignore
* [x] delete SVGs included in prior commits

### CODE REVIEW

**Review by csarven — DISMISSED** [review]
https://github.com/solid/notifications/pull/33#issuecomment-1029144741

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+*.mmd.svg
```
Comment (elf-pavlik) [inline .gitignore:2]: @csarven can you please suggest an alternative here? thanks!

Comment (csarven) [reply .gitignore:2]: Don't use Mermaid + GitHub actions.

**Comment by elf-pavlik** [comment]
I had moment to test drive it on my fork and after catching 2 glitches it works fine now

https://elf-pavlik.github.io/notifications/protocol

**Pull request merged**